### PR TITLE
add one json test data file

### PR DESCRIPTION
**Pull request by elias-shoursoh**
modify all test cases accordingly

### CODE REVIEW

File: data/tests_data.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "workspace_test": {
```
Comment (nirtal85) [inline data/tests_data.json:2]: remove test from each object

File: data/tests_data.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "workspace_name": "test",
+    "new_workspace_name": "another test",
```
Comment (nirtal85) [inline data/tests_data.json:4]: remove workspace from key

File: tests/conftest.py [hunk 3]
```diff
@@ -58,8 +58,10 @@ def create_driver(prep_properties, request):
 @pytest.hookimpl(tryfirst=True, hookwrapper=True)
 def pytest_runtest_makereport():
     outcome = yield
+    driver = dg.DRIVER
+    assert driver is not None, "Expected instance of a Web Driver but got None instead"
```
Comment (nirtal85) [inline tests/conftest.py:62]: why we need this?

File: tests/test_forgot_password.py [hunk 4]
```diff
@@ -6,33 +6,36 @@
...
-    _SUCCESS_MSG = "A reset link has been sent to the email address, if it has been used to register for an account."
-    _ERROR_MSG = "We can't find a user with that e-mail address."
     _DATA_FILE_NAME = "data.xlsx"
+    _JSON_FILE_NAME = "tests_data.json"
 
     @allure.description("Forgot password feature test with a valid email address")
     @allure.title("Forgot Password with valid email test")
     def test_valid_email(self, create_driver, prep_properties):
         config_reader = prep_properties
+        json_reader = JsonParser(self._JSON_FILE_NAME)
```
Comment (nirtal85) [inline tests/test_forgot_password.py:24]: can it be in a fixture?

**Review by nirtal85 — CHANGES_REQUESTED** [review]

File: tests/conftest.py (hunk 3 above)
Comment (elias-shoursoh) [reply tests/conftest.py:82]: to pass it to the hook method for screen shots.
I still don't have a solution for driver instance passing to that method

**Pull request merged**